src/mlpack/core/arma_extend/README.md

```diff
@@ -3,14 +3,8 @@ to still support older versions of Armadillo.  Therefore some files are licensed
 under the Mozilla Public License v2.0 (MPL2).
 
 These are the files under the MPL:
-
- - fn_ind2sub.hpp
- - SpMat_extra_bones.hpp
- - SpMat_extra_meat.hpp
- - operator_minus.hpp
- - hdf5_misc.hpp
- - Mat_extra_bones.hpp
- - Mat_extra_meat.hpp
+ 
+ * serialize_armadillo.hpp
```

Comment:
Actually your new `serialize_armadillo.hpp` implementations aren't under the MPL, they can be under BSD since you wrote them for mlpack.  The only files to include there are explicit backports of Armadillo functions that come from newer versions of Armadillo.

Update: I took a look at the other files in this directory... I think we're not actually backporting anything anymore!  If that's the case, we can remove this `README.md` file too. :+1:

Reply:
OK, perfect, I will remove the readme entirely